build/setup.sh

```diff
@@ -23,4 +15,36 @@ else
     apt update
     apt install -y $apt_dependencies
 fi
+if [ $? -ne 0 ]; then
```

Comment:
should we keep quoting around _$?_ consistent? Sometimes you use `if [ $? -ne 0 ];` then and others `if [ "$?" -ne "0" ];`